pyspark-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/nodes.py

```diff
@@ -0,0 +1,75 @@
+"""
+This is a boilerplate pipeline
+generated using Kedro {{ cookiecutter.kedro_version }}
+"""
+
+import logging
+from typing import Dict, Tuple
+
+import numpy as np
+import pandas as pd
+from pyspark.sql import DataFrame
+
+
+def split_data(data: DataFrame, parameters: Dict) -> Tuple:
+    """Splits data into features and targets training and test sets.
+
+    Args:
+        data: Data containing features and target.
+        parameters: Parameters defined in parameters.yml.
+    Returns:
+        Split data.
+    """
+
+    # Split to training and testing data
+    data_train, data_test = data.randomSplit(
+        weights=[parameters["train_fraction"], 1 - parameters["train_fraction"]]
+    )
+
+    X_train = data_train.drop(parameters["target_column"])
+    X_test = data_test.drop(parameters["target_column"])
+    y_train = data_train.select(parameters["target_column"])
+    y_test = data_test.select(parameters["target_column"])
+
+    print(y_train)
```

Comment:
Is this print line needed?

Reply:
oops, you are right it was leftover from debugging, thank you.